src/views/conference/2019/index/index.jsx

```diff
@@ -0,0 +1,348 @@
+                                    src="/svgs/conference/index/calendar-icon-solid.svg"
+                                />
+                            </td>
+                            <td><FormattedMessage id="conference-2019.date" /></td>
+                            <td>
+                                <FormattedDate
+                                    day="2-digit"
+                                    month="long"
+                                    value={new Date(2019, 4, 30)}
+                                    year="numeric"
+                                />
+                                {' - '}
+                                <FormattedDate
+                                    day="2-digit"
+                                    month="long"
+                                    value={new Date(2019, 4, 31)}
+                                    year="numeric"
+                                />
+                            </td>
+                        </tr>
+                        <tr className="conf2019-panel-row">
+                            <td className="conf2019-panel-row-icon">
+                                <img
+                                    alt="Map Icon"
+                                    className="conf2019-panel-row-icon-image"
+                                    src="/svgs/conference/index/map-icon-solid.svg"
+                                />
+                            </td>
+                            <td><FormattedMessage id="conference-2019.location" /></td>
+                            <td>{'Santiago, Chile'}</td>
+                        </tr>
+                        <tr className="conf2019-panel-row">
+                            <td className="conf2019-panel-row-icon">
+                                <img
+                                    alt="Audience Icon"
+                                    className="conf2019-panel-row-icon-image"
+                                    src="/svgs/conference/index/audience-icon-solid.svg"
+                                />
+                            </td>
+                            <td><FormattedMessage id="conference-2019.audience" /></td>
+                            <td><FormattedMessage id="conference-2019.chileAudience" /></td>
+                        </tr>
+                        <tr className="conf2019-panel-row">
+                            <td className="conf2019-panel-row-icon">
+                                <img
+                                    alt="Language Icon"
+                                    className="conf2019-panel-row-icon-image"
+                                    src="/svgs/conference/index/language-icon-solid.svg"
+                                />
+                            </td>
+                            <td><FormattedMessage id="conference-2019.language" /></td>
+                            <td>{'Español (Spanish) - simultaneous translation during plenary sessions'}</td>
```

Comment:
This should probably be translated, but we don't want `Español` translated, so I might use a placeholder:
`{espanol} (Spanish) - simultaneous translation during plenary sessions` 

We should also add information to the context in transifex so that people know how to translated it.

Is the translation just to English? Should probably include that if so.

Reply:
My thinks it's just into English; changing.

I'm going to just use "Spanish" for now, and it will read "Espanol" in Espanol.